### ISSUE TITLE
李宏毅机器学习-为什么训练会失败 · 核子的Blog

### ISSUE THREAD

**Issue by hezj-opt** [hezj-opt/comments#3]
https://hezj-opt.github.io/Deep-Learning/%E6%9D%8E%E5%AE%8F%E6%AF%85%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/%E6%9D%8E%E5%AE%8F%E6%AF%85%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0-%E4%B8%BA%E4%BB%80%E4%B9%88%E8%AE%AD%E7%BB%83%E4%BC%9A%E5%A4%B1%E8%B4%A5/ 

 Training loss很高

这时发生了什么
训练方法

Batch

Batch对训练效率影响
Batch对训练效果影响

Momentum
Learning rate
Loss function

Training data的loss很小

过拟合

调整模型复杂度
添加更多数据
